### DISCUSSION TITLE
My readme.md is not updating with any instructions

### DISCUSSION THREAD

**Discussion by LuckyLuc96** [skills/.github#537] [Introduction to GitHub]
Hi. I hope I'm not missing anything, but the instructions said that I should expect my readme.md to update 20 seconds after making the repo, but it has been about 40 minutes without any new instructions found. I could just be missing something obvious. 

https://github.com/LuckyLuc96/skills-introduction-to-github/tree/main?tab=readme-ov-file
![Screenshot from 2024-03-28 22-33-41](https://github.com/skills/.github/assets/94196417/ead3709a-d48f-4489-961e-27c60693230b)

**Comment by LuckyLuc96** [comment]
I see some stuff queued up in the workspace section. I think this is something to do with github having some lag so I suspect I can check back later to the readme being updated.

**Reply by jankeesdudel** [reply]
Same issue.

**Reply by potato-up** [reply]
Me too - but I stumbled across the instructions in the .github folder > steps (screenshots below). The _0.welcome.md_ didn't have anything in mine, but _1-create-a-branch.md_ down did. It also said that if I refresh the page it would automatically go to the next step but that didn't work for me so I just went back to the steps folder and manually went to the next step for each. 

Also, for step 3: Open a pull request - the pop up to create a pull request automatically may not appear under code, but in the pull requests tab (or at least for me) but either way, the manual instructions work.

Other than that all the instructions are easy to follow :)

Hope this helps. 
<img width="194" alt="Screenshot 2024-03-29 at 11 48 09 pm" src="https://github.com/skills/.github/assets/165435677/9e85bed0-0a9a-44db-a1c8-4d4bbf11bdd5">
<img width="469" alt="Screenshot 2024-03-29 at 11 49 46 pm" src="https://github.com/skills/.github/assets/165435677/41aa094f-eb6c-42aa-a5bb-fbb19dbe4117">
<img width="488" alt="Screenshot 2024-03-29 at 11 49 55 pm" src="https://github.com/skills/.github/assets/165435677/6384b71c-b74b-4bb5-b3ba-bf77cae0276a">
